pxtlib/sprite-editor/canvasGrid.ts

```diff
@@ -493,4 +509,20 @@ namespace pxtsprite {
             this.dead = true;
         }
     }
+
+    export interface ClientCoordinates {
+        clientX: number;
+        clientY: number;
+    }
+
+    function clientCoord(ev: PointerEvent | MouseEvent | TouchEvent): ClientCoordinates {
+        if ((ev as TouchEvent).touches) {
+            const te = ev as TouchEvent;
+            if (te.touches.length) {
+                return te.touches[0];
```

Comment:
can we handle all the touch events? If the user press both the dpath and A/B,  we get multiple entries in "touches" (future PR)